### PR TITLE
postTask explainer cleanup

### PR DESCRIPTION
**Pull request by shaseley**
In addition to some minor edits, the main changes are:
 
 * Add an explicit goals and non-goals section
 * Add metadata (author, issue tracker, spec)
 * Clean up introduction prose
 * Reorder and tweak post-MVP exploration
 * Mention API shape evolution

### CODE REVIEW

**Comment by shaseley** [comment]
I'd like to get this merged before filing the TAG spec review, then I'll circle back and revise the priority inversion content.

@natechapin can you PTAL?

/cc @jyasskin PTAL as well if you have cycles, specifically if the goals and non-goals look okay.

File: explainers/prioritized-post-task.md [hunk 1]
```diff
@@ -108,14 +152,13 @@ support these (see FAQ below).
    directly in response to user input, or updating in-viewport UI state.
 
 2. **user-visible**: User-visible tasks are those that will be visible to the
-   user, but not either not immediately or do not block the user from
-   interacting with the page. These tasks represent either a different
-   importance or timescale as user-blocking tasks.
+   user, but either not immediately or do not block the user from interacting
+   with the page. These tasks represent either a different importance or
+   timescale as user-blocking tasks.
```
Comment (natechapin) [inline explainers/prioritized-post-task.md:157]: s/as/from/ ?

**Review by natechapin — APPROVED** [review]
Just a grammar nit...

File: explainers/prioritized-post-task.md (hunk 1 above)
Comment (shaseley) [reply explainers/prioritized-post-task.md]: Oof, yeah that's a bit of an awkward sentence. I changed it slightly.

**Comment by shaseley** [comment]
Thanks @natechapin! I'm going to merge this now. @jyasskin if you find anything afterwards, I'd be happy to create a new PR.

**Pull request merged**